src/cake.py

```diff
@@ -3411,7 +3438,7 @@ def from_scanlines(cls, producer):
 
     def to_scanlines(self):
         def fmt(z, m):
-            return (z, m.vp, m.vs, m.rho, m.qp, m.qs)
+            return (z, m.vp, m.vs, m.rho, m.qp, m.qs, m.eta1, m.eta2, m.alpha)
```

Comment:
Won't this break code using `to_scanlines`, e.g. used in other fomosto backends? Adding a flag would be a possibility to keep the old behavior as default.

Reply:
Yes it would, which is why we would need to discuss how to handle it ...